blog/2016-10-24-whats-new-in-react-native.md

```diff
@@ -0,0 +1,86 @@
+---
+title: What's New in React Native v0.36
+author: Héctor Ramos
+authorTitle: Developer Advocate at Facebook
+authorURL: https://twitter.com/hectorramos
+authorImage: https://s.gravatar.com/avatar/f2223874e66e884c99087e452501f2da?s=128
+authorTwitter: hectorramos
+category: announcements
+---
+
+Today we've released [React Native v0.36](https://github.com/facebook/react-native/releases/tag/v0.36.0). You can upgrade to v0.36 by running the following commands in a terminal:
+
+```bash
+$ npm install --save react-native@0.36
+$ react-native upgrade
+```
+
+Read on to learn more about what's new in React Native v0.36.
+
+## Headless JS
+
+Headless JS is a way to run tasks in JavaScript while your app is in the background. It can be used, for example, to sync fresh data, handle push notifications, or play music.
+
+To get started, define your async task in a dedicated file (e.g. `SomeTaskName.js`):
+
+```javascript
+module.exports = async (taskData) => {
+  // Perform your task here.
+}
+```
+
+Next, register your task in on `AppRegistry`:
+
+```javascript
+AppRegistry.registerHeadlessTask('SomeTaskName', () => require('SomeTaskName'));
+```
+
+Using Headless JS does require some native Java code to be written in order to allow you to start up the service when needed. Take a look at our new [Headless JS docs](/react-native/docs/headless-js-android.html) to learn more!
+
+## Keyboard Component
```

Comment:
Is this a react component? If not, how about callign this section "The Keyboard API"